CRM/Case/Info.php

```diff
@@ -245,4 +245,40 @@ public static function onToggleComponents($oldValue, $newValue, $metadata) {
     }
   }
 
+  /**
+   * @return array
+   *   Array(string $value => string $label).
+   */
+  public static function getRedactOptions() {
+    return array(
+      'auto' => ts('Auto'),
```

Comment:
Well, we could call it "Magic", and it would make just as much sense. ;)

But... yes, "Default" does feel a bit more standard. Updating.